blog/posts/mootools-1-6-0-release.md

```diff
@@ -0,0 +1,71 @@
+#1.6.0 is out!
```

Comment:
Not the jekyll like header?

For example https://raw.githubusercontent.com/mootools/website/master/blog/posts/a-better-way-to-use-elements.md

has 

```
---
title: "A Better Way to use Elements"
date: "Fri, 19 Mar 2010 22:41:41 +0000"
author: "Sean McArthur"
tags: "all"
permalink: "2010/03/19/a-better-way-to-use-elements/"
---
```